memcached.c

```diff
@@ -3332,16 +3332,28 @@ static inline char *_ascii_get_suffix_buf(conn *c, int i) {
 // FIXME: the 'breaks' around memory malloc's should break all the way down,
 // fill ileft/suffixleft, then run conn_releaseitems()
 /* ntokens is overwritten here... shrug.. */
-static inline void process_get_command(conn *c, token_t *tokens, size_t ntokens, bool return_cas) {
+static inline void process_get_command(conn *c, token_t *tokens, size_t ntokens, bool return_cas, bool should_touch) {
```

Comment:
I can refactor this myself later, but better would be to pass in flags, then convert those to bools at the top of the function.